lms/djangoapps/courseware/date_summary.py

```diff
@@ -167,9 +134,7 @@ def date(self):
 
     @property
     def title(self):
-        return _(u'Today is {date}').format(
-            date=self.date.astimezone(self.time_zone).strftime(self.date_format.encode('utf-8')).decode('utf-8')
-        )
+        return 'Today is {date}'
```

Comment:
I see you are instead translating this in the mako template.  That will work.  However, returning an untranslated string here and then translating it elsewhere is not a standard pattern and difficult to read.  May I suggest returning a constant value here instead to indicate that the string should be "Today is" and then the string is created only in the mako template.